RNA-seq/05-nb_cell_line_DESeq2.Rmd

```diff
@@ -10,9 +10,9 @@ output:
 
 In this notebook, we'll perform an analysis to identify the genes that are differentially expressed in _MYCN_ amplified vs. nonamplified neuroblastoma cell lines. 
 
-These RNA-seq data are from [Harenza, et al. _Scientific Data._ 2017.](https://doi.org/10.1038/sdata.2017.33).
+These RNA-seq data are from [Harenza, _et al._ (2017)](https://doi.org/10.1038/sdata.2017.33).
 
-More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) from Love, Anders, and Huber from which this is adapted (see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8))
+More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) from Love, Anders, and Huber from which this is adapted (see also: [Love, _et al._ (2014)](https://doi.org/10.1186/s13059-014-0550-8)).
 
 DESeq2 takes unnormalized counts or estimated counts and does the following:
```

Comment:
This is a comment for 2018 Jaclyn as much as it is for you, but we since we're here and I expect you will be instructing for this material based on earlier conversations...

How much detail to go into here is always an area of uncertainty for me. It's good to mention the three steps, I think. Of them, I feel like the size factors bit might be the most important to talk about because it highlights some issues with RNA-seq data (e.g., library size and composition). On the other hand, we very well may not do it justice in this format and are better off linking to the HBC course that is intended to spend much more instruction time on DGE analysis. I don't know what the "right" answer is.

Reply:
Works for me 👍